libfastjson.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: libfastjson
+  version: 1.2304.0
+  epoch: 0
+  description: a fast json library for C
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libtool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3a8402c1de7c7747c95229db26d8d32fb85a7a52
+      repository: https://github.com/rsyslog/libfastjson
+      tag: v${{package.version}}
+
+  - runs: |
+      libtoolize --copy
+      aclocal -I m4 --install
+      autoheader
+      autoconf
+      automake --foreign --add-missing --force-missing --copy
+      ./configure --prefix=/usr
```

Comment:
I think this (these packages in general) might default to statically linking based on the alpine configure flag here: https://git.alpinelinux.org/aports/tree/main/libfastjson/APKBUILD#n30

I'm not sure though if the make command builds both the .so and .a files. I can check in a bit to confirm

Reply:
We do get the .so and .la files from this build, but we don't get the .a files...  Do we need those too, @jamonation ?
```

/usr/lib/libfastjson.so.4.3.0
/usr/lib/libfastjson.so.4
/usr/lib/libfastjson.la

```

Reply:
We should remove the *.la files, those are only useful for libtool

Reply:
We shouldn't need static libs, shared is fine

Reply:
Our `autoconf/make-install` pipeline removes them for us, marking as resolved - https://github.com/chainguard-dev/melange/blob/4ecc73b9b01f3a4f996836ccbc10dcac60ea3014/pkg/build/pipelines/autoconf/make-install.yaml#L26